apps/testingaccessibility/src/utils/format-prices-for-product.ts

```diff
@@ -52,18 +60,49 @@ export async function formatPricesForProduct({
   const pppDiscountPercent = getPPPDiscountPercent(country)
   const bulkCouponPercent = getBulkDiscountPercent(quantity)
 
-  const pppApplied = quantity === 1 && couponId && pppDiscountPercent > 0
-  const pppAvailable = quantity === 1 && pppDiscountPercent > 0
-  const bulkDiscountAvailable = bulkCouponPercent > 0
+  // if there's a coupon implied because an id is passed in, load it to verify
+  const {merchant_coupons_by_pk: appliedCoupon} = couponId
+    ? await getMerchantCoupon({
+        id: couponId,
+      })
+    : {merchant_coupons_by_pk: undefined}
+
+  const pppApplied =
+    quantity === 1 && appliedCoupon?.type === 'ppp' && pppDiscountPercent > 0
+
+  // pick the bigger discount during a sale
+  const appliedCouponLessThanPPP = appliedCoupon
+    ? appliedCoupon.percentage_discount < pppDiscountPercent
+    : true
+  const appliedCouponLessThanBulk = appliedCoupon
+    ? appliedCoupon.percentage_discount < bulkCouponPercent
+    : true
+
+  const pppAvailable =
+    quantity === 1 && pppDiscountPercent > 0 && appliedCouponLessThanPPP
+  const bulkDiscountAvailable =
+    bulkCouponPercent > 0 && appliedCouponLessThanBulk && !pppApplied
 
-  const defaultPriceProduct: FormattedPrice = {
+  let defaultPriceProduct: FormattedPrice = {
     ...noPricesProduct,
     quantity,
     unitPrice: firstPrice.unit_amount,
     calculatedPrice: firstPrice.unit_amount * quantity,
     availableCoupons: [],
   }
 
+  if (appliedCoupon?.type === 'site') {
+    defaultPriceProduct = {
+      ...defaultPriceProduct,
+      calculatedPrice: getCalculatedPriced({
+        unitPrice: defaultPriceProduct.unitPrice,
+        percentOfDiscount: appliedCoupon.percentage_discount,
+        quantity,
+      }),
+      appliedCoupon,
+    }
+  }
```

Comment:
if the couponId is passed in and it's `site` type, the default price is the price with that coupon applied